### ISSUE TITLE
Counter not applied to text fields

### ISSUE THREAD

**Issue by lukasbestle** [getkirby/panel#903]
Moved from #900 by @jakobploens:

> In v2.3.2, the counter isn't applied at all, for both max and maxLength.

**Comment by lukasbestle** [comment]
It's intended behavior that it isn't applied with the `max` validation, but the following code worked for me when implementing this:

```
fields:
  myfield:
    label: My field
    type: text
    validate:
      maxLength: 30
```

**Comment by jakobploens** [comment]
I see :) Yes, maxLength is working, but only with the custom stylesheet and the z-index fix from #900.
